src/components/Header.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  
+  <link rel="stylesheet" href="../styles/root.css">
+</head>
+<body>
+<!-------------------------Header Page Start------------------------->
+  
+<div class="ComponentMainBoxOne img-responsive">
+
+  <header class="headerContainerMain header-center Header-TextSize box-shadow quiz-container-bg">
+    <div class="headerContainerOne flex-evenly padding-small-left1">
+      <img class="logoImage" src="./Images/Phoenix UI.jpg" alt="">
+      <h4><a href="./HomePage.html" target="iframe-main-container">Vision Quiz</a></h4>
+    </div>
+
+    <div class="headerContainerTwo flex padding-top-normal1">
+      <form class="search-container-main" action="#">
+        <input class="boxShadow search-container" type="search" name="" placeholder="search" id="">
```

Comment:
if there is no need of id can remove that

Reply:
Okay, I'll do that.